### PR TITLE
profile  type enum added to profile model

### PR DESCRIPTION
**Pull request by chinexman**
### What this PR does (required):
- profile enum type added to profile model to determine the accountType, if the user is a pet Owner or a Pet Sitter

### CODE REVIEW

**Review by moffatethan — CHANGES_REQUESTED** [review]
Pausing review on this

PR Title is not good at all and the PR description isn't filled out.

File: server/db.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const mongoose = require("mongoose");
 
+console.log(process.env.MONGO_URI)
+
```
Comment (MUGABA) [inline server/db.js:4]: I would suggest you remove the debug logs

File: server/sample.env [hunk 2]
```diff
@@ -1,3 +1,3 @@
-JWT_SECRET=
-MONGO_URI=
+JWT_SECRET=abcd
+MONGO_URI=mongodb+srv://Bornagainm1:<password>@chinexman.jyiso.mongodb.net/test
```
Comment (MUGABA) [inline server/sample.env:2]: Would mind leaving these to the .env file

**Comment by chinexman** [comment]
i have effected the changes

File: server/db.js (hunk 1 above)
Comment (moffatethan) [reply server/db.js:4]: Did you happen to catch this console log mentioned?

**Review by moffatethan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**